tests/Block/BlockContextManagerTest.php

```diff
@@ -122,8 +128,7 @@ public function testWithInvalidSettings(): void
         ]);
         $block->expects(static::once())->method('getSetting')->with('template')->willReturn('custom.html.twig');
 
-        $cacheBlock = ['by_class' => [], 'by_type' => []];
-        $manager = new BlockContextManager($blockLoader, $serviceManager, $cacheBlock, $logger);
+        $manager = new BlockContextManager($blockLoader, $serviceManager, $logger);
```

Comment:
```suggestion
        $manager = new BlockContextManager($blockLoader, $serviceManager);
```
?

Reply:
no the 3rd argument as logger is fine and even required in this test. It performs assertions on method calls for the logger.